core/platform/search/gae_search_services.py

```diff
@@ -118,6 +130,20 @@ def _dict_to_search_document(d):
 
 
 def _make_fields(key, value):
+    """Returns a list of fields instantiated using given field name and
+       values.
+
+    Args:
+      - key: name of the field, a string.
+      - value: value of the field. Data type can be string, numeric type,
```

Comment:
is this values (with an s) or just one value? (this isn't consistent with line 134)